### PR TITLE
[Examples] Add Carla test env

### PR DESCRIPTION
**Pull request by ericl**
## What do these changes do?

This adds a simple gym wrapper around Carla. Note that the reward function is currently hard-coded to drive straight down the street.

You can run `env.py` to try out the environment, or `train.py` / `train_dqn.py` to attempt training.

### CODE REVIEW

File: examples/carla/train_ppo.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
```
Comment (pcmoritz) [inline examples/carla/train_ppo.py]: add the full` from future import *` thing here?

File: examples/carla/env.py [hunk 2]
```diff
@@ -0,0 +1,343 @@
+#!/usr/bin/env python3
+
+from __future__ import print_function
```
Comment (pcmoritz) [inline examples/carla/env.py]: also add the other lines?

File: examples/carla/train_dqn.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+from ray.tune import register_env, run_experiments
+
+from env import CarlaEnv, ENV_CONFIG
+
```
Comment (pcmoritz) [inline examples/carla/train_dqn.py]: from future import ...

**Comment by ericl** [comment]
Updated

**Review by pcmoritz — APPROVED** [review]

**Pull request merged**